packages/EasyCore/tests/Doctrine/Dispatchers/DeferredEntityEventDispatcherTest.php

```diff
@@ -1,359 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace EonX\EasyCore\Tests\Doctrine\Dispatchers;
-
-use EonX\EasyCore\Doctrine\Dispatchers\DeferredEntityEventDispatcher;
-use EonX\EasyCore\Doctrine\Events\EntityCreatedEvent;
-use EonX\EasyCore\Doctrine\Events\EntityUpdatedEvent;
-use EonX\EasyCore\Tests\AbstractTestCase;
-use Prophecy\Argument;
-use Psr\EventDispatcher\EventDispatcherInterface;
-use stdClass;
-
-/**
- * @covers \EonX\EasyCore\Doctrine\Dispatchers\DeferredEntityEventDispatcher
- */
-final class DeferredEntityEventDispatcherTest extends AbstractTestCase
```

Comment:
Why did you remove it?

Reply:
I did it for the following reasons:
1. Now `EntityEventSubscriberTest` is the integration test that covers all cases that were in `DeferredEntityEventDispatcherTest`.
2. It's difficult to maintain `DeferredEntityEventDispatcherTest` because there are a lot of implementation details.

But if you think that `DeferredEntityEventDispatcherTest` can catch some errors, I will return it.